src/main/java/org/takes/rs/RsWithType.java

```diff
@@ -58,6 +84,17 @@ public RsWithType(final Response res, final CharSequence type) {
      * Ctor.
```

Comment:
@essobedo can you explain here in more detail how using `charset` will impact header?